controllers/console_controller.go

```diff
@@ -0,0 +1,135 @@
+package controllers
+
+import (
+	"context"
+	"strings"
+
+	"github.com/go-logr/logr"
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
```

Comment:
can we separate imports like https://github.com/red-hat-storage/odf-operator/blob/main/controllers/subscriptions.go#L19